README.md

```diff
@@ -313,18 +350,19 @@ It has several usages and more will be added based on demand.
 
 It defines how this object will be handled. Each type has a different set of properties.
 
-##### Supported values
+##### Supported types
 - `NAME`: To be used as the name of the app.
 - `NAMESPACE`: To be used as the kubernetes namespace where the app will installed.
 - `IMAGE`: Link to a docker image.
-- `REPORTING_SECRET`: The Secret resource name containing the usage reporting credentials
 - `GENERATED_PASSWORD`: A value to be generated at deployment time, following common password requirements.
+- `REPORTING_SECRET`: The Secret resource name containing the usage reporting credentials
 - `SERVICE_ACCOUNT`: The name of a pre-provisioned k8s `ServiceAccount`. If it does not exist, one is created.
 - `STORAGE_CLASS`: The name of a pre-provisioned k8s `StorageClass`. If it does not exist, one is created.
+- `STRING`: A string to be used as password.
```

Comment:
Not just password. This type is intended for string manipulation. For now, the only feature we support is allowing the string to be used in places of manifests that require base 64 encoded values.